README.md

```diff
@@ -25,6 +26,15 @@ Kata Containers combines technology from [Intel® Clear Containers](https://gith
 
 Kata Containers is working to build a global, diverse and collaborative community. Anyone who is interested in supporting the technology is welcome to participate. We are seeking different expertise and skills, ranging from development, operations, documentation, marketing, community organization and product management.
 
+## Join Us
+
+You can join our community on any of several places:
+
+* [Mailing List](http://lists.katacontainers.io/)
+* IRC on IRC.freenode.net: [#kata-dev](http://webchat.freenode.net/?channels=kata-dev) and [#kata-general](http://webchat.freenode.net/?channels=kata-general)
```

Comment:
Join discussions in the [#kata-dev](http://webchat.freenode.net/?channels=kata-dev) and [#kata-general] rooms using IRC on IRC.freenode.net.